.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,7 @@
 | Bug fix?      | yes/no
 | New feature?  | yes/no <!-- don't forget to update src/**/CHANGELOG.md files -->
 | BC breaks?    | no     <!-- see https://symfony.com/bc -->
-| Deprecations? | yes/no <!-- don't forget to update UPGRADE-*.md files -->
+| Deprecations? | yes/no <!-- don't forget to update UPGRADE-*.md and CHANGELOG.md files -->
```

Comment:
This is ambiguous as people must not change the `CHANGELOG-*.md` in the root directory, but only the ones from the bridge/bundle/component directories.

Reply:
Indeed, updated as "New feature?" with `src/**/CHANGELOG.md`.